src/query/service/src/interpreters/access/privilege_access.rs

```diff
@@ -625,50 +673,112 @@ impl AccessChecker for PrivilegeAccess {
 
             // Database.
             Plan::ShowCreateDatabase(plan) => {
-                self.validate_db_access(&plan.catalog, &plan.database, UserPrivilegeType::Select, false).await?
+                self.validate_db_access(
+                    &plan.catalog,
+                    &plan.database,
+                    UserPrivilegeType::Select,
+                    false,
+                )
+                .await?
             }
             Plan::CreateDatabase(_) => {
                 self.validate_access(&GrantObject::Global, UserPrivilegeType::CreateDatabase)
                     .await?;
             }
+            // Task
+            // Is it compatible with the permission check of the old version?
             Plan::CreateTask(_) => {
-                self.validate_access(&GrantObject::Global, UserPrivilegeType::CreateTask)
-                    .await?;
+                if self
```

Comment:
I think it's not necessary to add this check. cc @flaneur2020 @ZhiHanZ

Reply:
why? we need CREATE TASK or SUPER priviledge to create any task, right?

Reply:
In old query version create/drop/alter/execute/show/desc task need SUPER privilege.

But in this pr we add CREATE TASK privilege to control create task, and we add task ownership to control desc/show/execute task.

Add drop to control drop task, alter to control alter task.